frontend/app/(tabs)/create.tsx

```diff
@@ -13,42 +14,66 @@ import {
   H5,
   Text,
   TextArea,
-  View,
+  Switch,
 } from 'tamagui';
-import type { TabsContentProps } from 'tamagui';
 
 export default function Create() {
   const [isEnabled, setEnabledElements] = useState(false);
 
   return (
     <View>
       <YStack>
-        <H4 style={{ alignSelf: 'center' }}>NEW POST</H4>
-        <Tabs
-          defaultValue='tab1'
-          orientation='horizontal'
-          flexDirection='column'
-          width={400}
-          height={150}
-          borderRadius='$4'
-          borderWidth='$0.25'
-          overflow='hidden'
-          borderColor='$borderColor'
-        >
-          <Tabs.List
-            disablePassBorderRadius='bottom'
-            aria-label='Creating a New Post'
+        <H4 style={{ alignSelf: 'center', color: 'black' }}>NEW POST</H4>
+        <XStack>
+          <Text color={'black'} fontSize='$5' paddingStart='$5'>
+            is this a recipe?
+          </Text>
+          <Switch
+            marginLeft='$2'
+            size='$3'
+            onCheckedChange={() => setEnabledElements(!isEnabled)}
           >
-            <Tabs.Tab flex={1} value='tab1'>
-              <SizableText fontFamily='$body'>Byte</SizableText>
-            </Tabs.Tab>
-            <Tabs.Tab flex={1} value='tab2'>
-              <SizableText fontFamily='$body'>Recipe</SizableText>
-            </Tabs.Tab>
-          </Tabs.List>
-          <Tabs.Content value='tab1'>
+            <Switch.Thumb animation='bouncy' />
+          </Switch>
+        </XStack>
+        {!isEnabled ? (
+          <YStack>
+            <TextArea
+              placeholder={'Description...'}
+              multiline={true}
+              style={{
+                height: 130,
+                borderRadius: 5,
+                paddingHorizontal: 10,
+                textAlignVertical: 'top',
+              }}
+            />
+            <TextArea
+              placeholder={'List Steps...'}
+              multiline={true}
+              style={{
+                height: 130,
+                borderRadius: 5,
+                paddingHorizontal: 10,
+                textAlignVertical: 'top',
+              }}
+            />
+            <TextArea
+              placeholder={'List Ingredients...'}
+              multiline={true}
+              style={{
+                height: 130,
+                borderRadius: 5,
+                paddingHorizontal: 10,
+                textAlignVertical: 'top',
+              }}
+            />
+          </YStack>
+        ) : null}
+        {isEnabled ? (
+          <YStack>
             <TextArea
-              placeholder={'Write a Caption...'}
+              placeholder={'Upload IMG...'}
```

Comment:
For images, I'm not sure where to go but on my `profile.tsx` file there is something that uses the camera app to upload images.